tools/mmp/driver.cs

```diff
@@ -771,12 +773,35 @@ static void Pack (IList<string> unprocessed)
 			if (App.LinkMode != LinkMode.All && App.RuntimeOptions != null)
 				App.RuntimeOptions.Write (App.AppDirectory);
 
+			if (experimental_xm_aot.HasValue && experimental_xm_aot.Value && IsUnified) {
+				AOTBundle ();
+				Watch ("AOT Compile", 1);
+			}
+
 			if (!string.IsNullOrEmpty (certificate_name)) {
 				CodeSign ();
 				Watch ("Code Sign", 1);
 			}
 		}
 
+		static void AOTBundle ()
+		{
+			string monoExe = Is64Bit ? "/usr/local/bin/mono64" : "/usr/local/bin/mono";
```

Comment:
I think `/Library/Frameworks/Mono.framework/Commands` is the supported location of the mono binaries.

Also I'd use `mono32` and `mono64`, since I think the mono binary one day will switch to be 64-bit.